public/index.html.erb

```diff
@@ -142,6 +142,7 @@
             </div>
             <div class="col-lg-4 col-md-4">
               <p class="fork small-text text-center"><a href="https://github.com/dblock/slack-amber-alert" target="_blank">fork me on github</a> | <a href='/api'>RESTful API</a></p>
+ <p class="fork small-text text-center"><a href="privacy.html" target="_blank">Privacy Policy</a></p>
```

Comment:
I use a macro to make HTML well-aligned. This caught a lot of issues like unclosed tags before, lets try to keep the HTML indented properly?